Bottom: 7a66247900e262504757d8b5d6099ec9e8fc22b3
Top:    a7d72866440920641e010f105f3ec8d0ed2755e2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-06-24 13:53:25 +0100

Refresh of reduce-tolerances-a-smidgin

---

diff --git a/src/netclass.c b/src/netclass.c
index 54c6d6e..6003f1b 100644
--- a/src/netclass.c
+++ b/src/netclass.c
@@ -31,16 +31,16 @@ static char *netclass_names[10] =
 
 static Coord clearances[10][10] =
   {
-    {MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3)}
+    {MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3)}
   };
